CRM/Upgrade/Incremental/Base.php

```diff
@@ -182,6 +182,51 @@ public static function addColumn($ctx, $table, $column, $properties, $localizabl
   }
 
   /**
+   * Modify a column in a table
+   *
+   * @param CRM_Queue_TaskContext $ctx
+   * @param string $table
+   * @param string $column
+   * @param string $properties
+   * @param bool $localizable is this a field that should be localized
+   * @param string $newColumn (new name for column)
+   * @return bool
+   */
+  public static function alterColumn($ctx, $table, $column, $properties, $localizable = FALSE, $newColumn = NULL) {
```

Comment:
@seamuslee001 you are more on top of where we are at with these upgrade helpers - how does this fit in?